src/ServiceInsight/Shell/About.xaml

```diff
@@ -48,14 +48,14 @@
 
                 <TextBlock Style="{StaticResource InfoText}">
                     <Run Text="Version  " />
-                    <Run Text="{Binding AppVersion}" />
+                    <Run Text="{Binding AppVersion, Mode=OneWay}" />
                     <Run Text="  Commit hash:  " />
-                    <Run Text="{Binding CommitHash}" />
+                    <Run Text="{Binding CommitHash, Mode=OneWay}" />
                 </TextBlock>
 
                 <TextBlock Style="{StaticResource InfoText}" Visibility="{Binding IsSplash, Converter={StaticResource BoolToVisibilityConverterInverted}}">
```

Comment:
`IsSplash` is also read-only but this binding has not been changed to `OneWay`. Is this because the binding is being used for `Visibility` rather than `Text`? (Same question for `ShellViewModel.StatusBarManager.ErrorMessageVisible`.)

I also noticed that `CopyrightText` is also read-only but the binding has not been changed to `OneWay`. Is this because `CopyrightText` is bound to a `TextBlock` instead of a `Run`?